fields.yml

```diff
@@ -203,6 +203,14 @@
             IP address of the destination.
     
             Can be one or multiple IPv4 or IPv6 addresses.
+        - name: cidr
+          type: ip
```

Comment:
Each `.cidr` field should actually be of type `ip_range` [1], not `ip` [2]. You can search an `ip` field with a CIDR range, but you can't store a range directly in it.

If you want to play with these in a Kibana console:

```JSON
PUT ip_test
{ "mappings": {
    "_doc": {
      "properties": {
        "address": {
          "type": "ip"
        },
        "range": {
          "type": "ip_range"
}}}}}

PUT ip_test/_doc/1
{ "address": "127.0.0.1" }
PUT ip_test/_doc/2
{ "address": "::1" }
PUT ip_test/_doc/3
{ "address": "255.255.255.0" }
# Fails:
PUT ip_test/_doc/4
{ "address": "10.0.0.0/8" }
PUT ip_test/_doc/4
{ "range": "10.0.0.0/8" }
# Fails:
PUT ip_test/_doc/5
{ "range": "255.255.255.0" }

GET ip_test/_search
DELETE ip_test
```



* [1] https://www.elastic.co/guide/en/elasticsearch/reference/current/ip.html
* [2] https://www.elastic.co/guide/en/elasticsearch/reference/current/range.html